examples/vanilla-es6/package.json

```diff
@@ -1,14 +1,18 @@
 {
   "private": true,
   "scripts": {
-    "compile": "java -jar node_modules/google-closure-compiler/compiler.jar -O ADVANCED --language_in=ES6_STRICT --new_type_inf --js_output_file='dist/bundle.js' 'src/**.js' -W VERBOSE",
```

Comment:
if you drop advanced no renaming will happen ... no need to switch to webpack :)

Reply:
Yeah.. I was really hoping to avoid a java install :(. And I think it is safe to say most people are using webpack/babel and not closure. So a switch will make this easier for most to maintain. No?